src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(
+    movie => movie.title.toLowerCase().includes(query.toLowerCase())
```

Comment:
move query.toLowerCase() in the variable, also use trim, because now I can search just spaces